api/src/index.js

```diff
@@ -3,26 +3,69 @@ import express from "express";
 import cors from "cors";
 import bodyParser from "body-parser";
 import knex from "./database_client.js";
-import nestedRouter from "./routers/nested.js";
+// import nestedRouter from "./routers/nested.js";
 
 const app = express();
 app.use(cors());
 app.use(bodyParser.json());
 
-const apiRouter = express.Router();
+const apiRouter = express.Router()
 
-// You can delete this route once you add your own routes
-apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
+apiRouter.get("/all-meals", async (req, res) => {
+  try {
+    const meals = await knex.raw("SELECT * FROM Meal");
```

Comment:
MySQL orders the items by ID by default. However, it would be a good idea to still order the items by ID in your query, just so we do not rely on default behaviour that might change.